api/metrics-api-jaxrs/src/main/java/org/hawkular/metrics/api/jaxrs/MetricsServiceLifecycle.java

```diff
@@ -399,7 +399,7 @@ private void startMetricsService() {
             if (metricsReportingHostname == null) {
                 metricNameService = new MetricNameService(adminTenant);
             } else {
-                metricNameService = new MetricNameService(metricsReportingHostname, adminTenant);
+                metricNameService = new MetricNameService(metricsReportingHostname);
```

Comment:
We need the admin tenant here to make sure metrics are stored with the correct tenant.